rocketpy/simulation/flight.py

```diff
@@ -2426,6 +2430,26 @@ def min_stability_margin(self):
         """Minimum stability margin."""
         return self.stability_margin(self.min_stability_margin_time)
 
+    @property
+    def initial_stability_margin(self):
+        """Stability margin at time 0.
+
+        Returns
+        -------
+        float
+        """
+        return self.stability_margin(0)
```

Comment:
Is the `t = 0` always a appropriate for getting the simulation beginning?

What if my simulation has an `initial_solution` (another `Flight` or a state vector)?

Reply:
Nice catch,
the solution was really simple: [ee01e9d](https://github.com/RocketPy-Team/RocketPy/pull/572/commits/ee01e9dd9f159d463e642465244ebe08f4038565)

Tests are passing.